rb/lib/selenium/webdriver/remote/response.rb

```diff
@@ -104,11 +103,11 @@ def add_backtrace(ex)
         end
 
         def status
-          @payload['status']
+          @payload['status'] || @payload['error']
```

Comment:
Yeah, it's 'error' & 'message' - http://www.w3.org/TR/webdriver/#processing-model
There are several things in this Response class that are handled differently between implementations, but I don't see how to address that right now. Something to consider down the road.